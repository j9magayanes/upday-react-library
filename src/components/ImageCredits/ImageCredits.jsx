import React from 'react';
import './imageCredits.css'

export const ImageCredits = ({ ...props}) => {
    return (
      <span className="imageCredits">Baden-Württemberg, Ulm: Der Angeklagte wird in Hand- und Fußschellen in den Gerichtssaal geführt. Foto: Felix Kästle/dpa</span>
    )
}