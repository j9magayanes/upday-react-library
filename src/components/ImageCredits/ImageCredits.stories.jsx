import React from 'react';

import { ImageCredits } from './ImageCredits';


export default {
  title: 'Form/ImageCredits',
  component:ImageCredits,
};

const Template = () =><ImageCredits  />;


export const Default = Template.bind({});