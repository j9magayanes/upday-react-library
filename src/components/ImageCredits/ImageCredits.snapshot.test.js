import React from 'react';
import renderer from 'react-test-renderer';
import { ImageCredits } from './ImageCredits';

test('renders ImageCredits component correctly', () => {
  const component = renderer.create(<ImageCredits />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
