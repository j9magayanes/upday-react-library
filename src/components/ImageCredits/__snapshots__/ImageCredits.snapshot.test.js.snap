// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ImageCredits component correctly 1`] = `
<span
  className="imageCredits"
>
  Baden-Württemberg, Ulm: Der Angeklagte wird in Hand- und Fußschellen in den Gerichtssaal geführt. Foto: Felix Kästle/dpa
</span>
`;
